File: AgentApp/Frontend/rent-ng-app/src/app/model/carModel.ts
/* tslint:disable:whitespace */
export class CarModel{
  id: number;
  isRemoved: boolean;
  modelName: string;

  constructor(id: number, isRemoved: boolean, name: string){
    this.id =id;
    this.isRemoved=isRemoved;
    this.modelName=name;
  }
}

File: AgentApp/Backend/src/main/java/rent/app/model/CarModel.java
package rent.app.model;

import lombok.Data;

import javax.persistence.*;
import javax.validation.constraints.Max;
import javax.validation.constraints.Min;

@Data
@Entity
@Table(name="CarModel")
public class CarModel {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column
    private String modelName;

    @Column
    private boolean isRemoved;

}
